### DISCUSSION TITLE
how to start it without an adapter, for vanilla node applications?

### DISCUSSION THREAD

**Discussion by patrickkmatias** [axe-me/vite-plugin-node#104] [Ideas]
I'm writing an app without any backend framework, how I do its setup?

**Comment by lbguilherme** [comment]
Config:

```
adapter: () => {},
```

And in your index file:

```
export const viteNodeApp = {};
```

**Comment by axe-me** [comment]
see https://github.com/axe-me/vite-plugin-node/blob/main/examples/node/app.ts
